packages/x-date-pickers-pro/src/DateTimeRangePicker/DateTimeRangePickerTabs.tsx

```diff
@@ -0,0 +1,242 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { styled, useThemeProps } from '@mui/material/styles';
+import composeClasses from '@mui/utils/composeClasses';
+import useEventCallback from '@mui/utils/useEventCallback';
+import { TimeIcon, DateRangeIcon, ArrowLeftIcon, ArrowRightIcon } from '@mui/x-date-pickers/icons';
+import { DateOrTimeViewWithMeridiem } from '@mui/x-date-pickers/internals/models';
+import { useLocaleText } from '@mui/x-date-pickers/internals/hooks/useUtils';
+import {
+  BaseTabsProps,
+  ExportedBaseTabsProps,
+} from '@mui/x-date-pickers/internals/models/props/tabs';
+import { isDatePickerView } from '@mui/x-date-pickers/internals/utils/date-utils';
```

Comment:
Will not be supported by some bundlers, should be re-exported from `@mui/x-date-pickers/internals`